sql/telemetry_derived/main_summary_v4/part1.sql

```diff
@@ -125,13 +125,11 @@ function ifnull(value1, value2) {
   }
   return value2;
 }
-
```

Comment:
Using persistent UDFs in `part1` and `part2` of `main_summary` resulted in failed tests because the UDFs could not be found. I'll look into this later, for now all the UDFs used in those 2 queries are temporary UDFs.